edb/pgsql/resolver/expr.py

```diff
@@ -117,6 +126,40 @@ def resolve_column_kind(
         case context.ColumnStaticVal(val=val):
             # special case: __type__ static value
             return _uuid_const(val)
+        case context.ColumnComputable(pointer=pointer):
+
+            expr = pointer.get_expr(ctx.schema)
+            assert expr
+
+            source = pointer.get_source(ctx.schema)
+            assert isinstance(source, s_types.Type)
+            # assert isinstance(source, (s_types.Type, s_pointers.Pointer))
```

Comment:
Stray

Reply:
This was left here because I was preparing to support link properties on computeds.

Can you confirm that we don't support them in EdgeQL?